samples/XAMLCatalog/XAMLCatalog/UIKitControls/UIButtonWithControlsViewController.m

```diff
@@ -64,9 +64,35 @@ - (UIControlState)getControlState:(NSString*)stateString {
     return state;
 }
 
+- (UIEdgeInsets)getUIEdgeInsetsFromString:(NSString*)edgeInsetsString {
+    UIEdgeInsets ret = UIEdgeInsetsZero;
+
+    if ([edgeInsetsString length] > 0) {
+        // Strip off curly braces and use the delimiter to separate components
```

Comment:
Adding comments to explain the format - we still need to build more intuitive controls but this will have to do for now.

---
In reply to: [104816170](https://github.com/Microsoft/WinObjC/pull/2156#discussion_r104816170) [](ancestors = 104816170)